### PR TITLE
Add `redis` and `redis-mixin`

### PR DESCRIPTION
**Pull request by colega**
This is some jsonnet used internally at Grafana Labs to operate redis, it also includes the mixin with a dashboard, alerts and some runbooks.

### CODE REVIEW

File: redis/config.libsonnet [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  _config+: {
+    namespace: error 'must specify namespace',
+    redis+: {
+      password: error 'must specify password',
+      port: 6379,
+      sentinel_port: 26379,
+      replicas: 3,
+      down_after_milliseconds: 15000,
+      diskSize: '50Gi',
```
Comment (jesusvazquez) [inline redis/config.libsonnet:10]: Is this a sane default? Seems huge to me for a start.

**Review by jesusvazquez — APPROVED** [review]
LGTM

File: redis/config.libsonnet (hunk 1 above)
Comment (colega) [reply redis/config.libsonnet:10]: It's opinionated, yes... but anyone can change it.

**Comment by colega** [comment]
Thank you for the review.

**Pull request merged**